src/scripts/clipperUI/panels/dialogPanel.tsx

```diff
@@ -55,14 +55,12 @@ export abstract class DialogPanelClass extends ComponentBase<{}, DialogPanelProp
 					<div id={Constants.Ids.dialogButtonContainer}>
 						{this.props.buttons.map((button, i) => {
 							return (
-								<a id={button.id} className="dialogButton" {...this.enableInvoke({callback: button.handler, tabIndex: 70})}>
-									<div className="wideButtonContainer">
-										<a className="wideButtonFont wideActionButton"										
-											style={Localization.getFontFamilyAsStyle(buttonFontFamily)}>
-											{ button.label }
-										</a>
-									</div>
-								</a>
+								<div id={button.id} className="wideButtonContainer dialogButton" {...this.enableInvoke({callback: button.handler, tabIndex: 70})}>
```

Comment:
Would this break tabbing+enter interactions now that enableInvoke is living on the outer div and not the a?

Reply:
Also wouldn't this break narrator?

Reply:
This definitely did not break tabbing+enter interactions, but I will test with Edge + Narrator

Reply:
This did break Edge + Narrator :(. Nice catch. I will go back to the "a"